app/Home/index.tsx

```diff
@@ -0,0 +1,52 @@
+"use client";
+
+import { ChangeEvent, useState } from "react";
+import Image from "next/image";
+import { Inter } from "next/font/google";
+import { NextPage } from "next";
+
+import styles from "./Home.module.css";
+
+import SPINNER from "../../public/assets/imgs/loading-spin.svg";
+import SearchBar from "@/components/SearchBar/SearchBar";
+import TableSection from "@/components/TableSection";
+
+const inter = Inter({ subsets: ["latin"] });
+
+const Home: NextPage = () => {
+  const [searchText, setSearchText] = useState<string>("");
+  const [currentState, setCurrentState] = useState<string>("");
+  const handleOnChangeEvent = (e: ChangeEvent<HTMLInputElement>) => {
+    setSearchText(e.target.value);
+  };
+  const handleOnChangeTypeEvent = () => {
+    console.log("handleOnChangeTypeEvent ----");
+  };
+  console.log(process.env.NEXT_PUBLIC_API_KEY);
+  return (
+    <main className={styles.container}>
+      <h1>Holidays across the world</h1>
+      <div className={`${styles.flexCenter} ${styles.marginTop}`}>
+        <SearchBar
+          id={"search-box"}
+          type={"text"}
+          name={"search-box"}
+          value={searchText}
+          placeHolder={"Search country"}
+          currentState={currentState}
+          onChangeEvent={handleOnChangeEvent}
+          onChangeTypeEvent={handleOnChangeTypeEvent}
+        />
+        <div>{searchText}</div>
+      </div>
+      <TableSection tableData={{}} currentState={currentState} />
```

Comment:
tableData. I am planning to implement it laster because I have not finished Fetch Data. About component design, I still do not know how I can design it better

Reply:
Okay, when you implement `tableData` I can help you understand better how to improve the component design system